templates/engage/_article-card.html

```diff
@@ -6,7 +6,7 @@ <h5 class="p-muted-heading u-no-margin--bottom">
   </header>
   <div class="blog-p-card__content">
     <h4>
-      <a href="{{title_link}}" >{{title | safe}}</a>
+      <a href="{%if active == 'false' and preview != None %}{{title_link}}?preview{% else %}{{title_link}}{% endif %}">{{title | safe}}</a>
```

Comment:
Should be a space `{%if`

Reply:
Space added.

Reply:
Done.

Reply:
I added an extra assertion too to check that the active pages do not get the `?preview` added.